From a78094c62824e65135f144ba4618df63f3530321 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Sat, 19 Feb 2011 18:56:00 +0000
Subject: [PATCH 03/72] t/test-lib.sh: add always-set NO_GETTEXT_POISON prerequisite
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add a new test prerequisite called NO_GETTEXT_POISON to t/test-lib.sh
that's always set for now.

Some of the gettextize patches that'll follow this one will change
their test outputs if a functioning gettext implementation with poison
support is sitting on top of them, but for now I only have harmless
no-op wrappers in gettext.h so NO_GETTEXT_POISON can be set all the
time.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 t/test-lib.sh |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/t/test-lib.sh b/t/test-lib.sh
index 0fdc541..ffcfa2e 100644
--- a/t/test-lib.sh
+++ b/t/test-lib.sh
@@ -1079,6 +1079,11 @@ esac
 test -z "$NO_PERL" && test_set_prereq PERL
 test -z "$NO_PYTHON" && test_set_prereq PYTHON
 
+# Always set the NO_GETTEXT_POISON prerequisite while we don't have a
+# real gettext implementation in Git. This will be replaced by
+# something that varies in a future patch series.
+test_set_prereq NO_GETTEXT_POISON
+
 # test whether the filesystem supports symbolic links
 ln -s x y 2>/dev/null && test -h y 2>/dev/null && test_set_prereq SYMLINKS
 rm -f y
-- 
1.7.2.3

